src/nyc_trees/apps/survey/models.py

```diff
@@ -40,6 +41,11 @@ class Survey(NycModel, models.Model):
                                  related_name='surveys_as_teammate',
                                  blank=True)
     is_flagged = models.BooleanField(default=False)
+    has_trees = models.BooleanField(default=True)
+    is_mapped_in_blockface_polyline_direction = models.BooleanField(
+        default=True)
+    is_left_side = models.BooleanField(default=True)
```

Comment:
I think we should remove the `default=True` for all 3 of these fields.

If we don't get the information for these fields from the client, it's an error.
I'd rather catch the bug earlier by failing validation during testing, than just silently using an incorrect default value.